local-config/config.yaml

```diff
@@ -0,0 +1,10 @@
+http:
+  prefix: someprefix
+maxWaitTimeForTrace: 10s
+postgresConnString: "host=127.0.0.1 user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+jaegerConnectionConfig:
+  endpoint: localhost:16685
+  tls:
+    insecure: true
+googleAnalytics:
+  enabled: false
```

Comment:
This config file is in an old format. Take a look at the new format: https://github.com/kubeshop/tracetest/blob/main/local-config/config.tests.yaml

Reply:
We are not even using this file so I just deleted it 😅